src/parseTorrent.ts

```diff
@@ -110,6 +110,9 @@ export class Metafile {
 	static decode(buf: Buffer) {
 		return new Metafile(bencode.decode(buf));
 	}
+	static getFileSystemSafeName(name: string) {
```

Comment:
I don't think this needs to be static, it can be an instance method—then we can return `this.name.replace(...)`

Reply:
Updated.